python/cugraph/tests/test_k_truss_subgraph.py

```diff
@@ -79,21 +77,13 @@ def compare_k_truss(graph_file, k, ground_truth_file, directed=True):
     return True
 
 
-DATASETS = [('../datasets/polbooks.csv',
-             '../datasets/ref/ktruss/polbooks.csv'),
-            ('../datasets/netscience.csv',
-             '../datasets/ref/ktruss/netscience.csv')]
-
-
-@pytest.mark.parametrize('graph_file, nx_ground_truth', DATASETS)
+@pytest.mark.parametrize('graph_file, nx_ground_truth', utils.DATASETS_4K)
 def test_ktruss_subgraph_DiGraph(graph_file, nx_ground_truth):
```

Comment:
This function has been removed in master because ktruss is not expected to run on directed graph.

Reply:
updated